src/qt/transactionview.h

```diff
@@ -77,6 +77,8 @@ class TransactionView : public QWidget
     QDateTimeEdit *dateTo;
     QAction *abandonAction;
     QAction *bumpFeeAction;
+    QAction *copyAddressAction;
```

Comment:
prefer `QAction* copyAddressAction{nullptr};` here and below and drop changes to constructor initializer list.